src/.schema.sql

```diff
@@ -1,15 +1,18 @@
-CREATE SCHEMA IF NOT EXIST kubercade AUTHORIZATION jxzxrgmv
+/* Note: this schema is PostgreSQL-specific and uses `timestamptz`. */
+CREATE SCHEMA IF NOT EXISTS kubercade;
 
-  CREATE TABLE IF NOT EXISTS kubercade.high_score_table (
-    game_index  integer,
-    name        varchar(80),
-    score       integer NOT NULL,
-    datetime    varchar(40) NOT NULL
-  );
+CREATE TABLE IF NOT EXISTS kubercade.high_score_table (
+  game_index  integer,
+  name        varchar(80) NOT NULL,
+  score       integer NOT NULL,
+  datetime    timestamptz NOT NULL,
+  PRIMARY KEY (game_index, score, datetime)
+);
 
-  CREATE TABLE IF NOT EXISTS kubercade.chat_table (
-    game_index  integer,
-    name        varchar(80),
-    message     varchar(400) NOT NULL,
-    datetime    varchar(40) NOT NULL
-  );
+CREATE TABLE IF NOT EXISTS kubercade.chat_table (
+  game_index  integer,
+  name        varchar(80) NOT NULL,
+  message     varchar(400) NOT NULL,
+  datetime    timestamptz NOT NULL,
+  PRIMARY KEY (game_index, score, datetime)
```

Comment:
You probably don't want `score` here, since it's not in the table.

Reply:
Done https://github.com/knative-portability/Kubercade/pull/22/commits/f75da7b85109365159369a7b2f36bcda42128328